ui/src/app/shared/help-icon.tsx

```diff
@@ -0,0 +1,11 @@
+import {Tooltip} from 'argo-ui/src/app/shared/components/tooltip/tooltip';
```

Comment:
Move help-icon and revision-help-icon to `argo-ui/src/app/shared/components` and import both using index file


```
import {RevisionHelpIcon} from '../shared/components'
```

Reply:
https://github.com/argoproj/argo-ui/pull/43